### PR TITLE
Fix broken documentation links

### PR DESCRIPTION
**Pull request by paulo-ferraz-oliveira**
# Description

Just a few broken links fixed, including the one in this template.

- [x] I have performed a self-review of my changes
- [x] I have read and understood the [contributing guidelines](/kerl/kerl/blob/master/CONTRIBUTING.md)

### CODE REVIEW

File: zsh_completion/_kerl [hunk 1]
```diff
@@ -4,7 +4,7 @@
 # Description
 # -----------
 #
-#  Completion script for kerl (http://github.com/spawngrid/kerl
+#  Completion script for kerl (https://github.com/spawngrid/kerl)
```
Comment (paulo-ferraz-oliveira) [inline zsh_completion/_kerl:7]: Has no real "influence" apart from showing up in searches.

Comment (jadeallenx) [reply zsh_completion/_kerl:7]: might as well make it `kerl/kerl` then

Comment (paulo-ferraz-oliveira) [reply zsh_completion/_kerl:7]: Sure. I didn't want to change it, on purpose, for history reasons.

Comment (paulo-ferraz-oliveira) [reply zsh_completion/_kerl]: 7f5f892

**Review by jadeallenx — APPROVED** [review]

**Comment by paulo-ferraz-oliveira** [comment]
This will get released in 4.2.0.

**Pull request merged**